README.md

```diff
@@ -129,6 +129,19 @@ SDK also includes a Metadata API to interact with AppData documents and IPFS CID
   // Decode AppData Hex to CID
   const decodedAppDataHex  = await cowSdk.metadataApi.appDataHexToCid(hash)
   console.log(decodedAppDataHex) //QmUf2TrpSANVXdgcYfAAACe6kg551cY3rAemB7xfEMjYvs
+
+  // Create an AppData Document
+    const appDataDoc = cowSdk.metadataApi.generateAppDataDoc({})
+    /* {
+        version: '0.1.0',
+        appCode: 'CowSwap',
+        metadata: {},
+      } */
```

Comment:
Could you add some instructions regarding what data/parameters this method expects?